temporal-sdk/src/main/java/io/temporal/internal/sync/ExternalWorkflowInvocationHandler.java

```diff
@@ -20,7 +20,7 @@ public ExternalWorkflowInvocationHandler(
       WorkflowExecution execution,
       WorkflowOutboundCallsInterceptor workflowOutboundCallsInterceptor,
       Functions.Proc1<String> assertReadOnly) {
-    this.workflowMetadata = POJOWorkflowInterfaceMetadata.newInstance(workflowInterface);
+    this.workflowMetadata = POJOWorkflowInterfaceMetadata.newInstance(workflowInterface, false);
```

Comment:
Got a bit confused on this trailing param reading:

```
if true, methods of the {@code anInterface} that are
annotated with {@link WorkflowMethod}, {@link QueryMethod}, {@link UpdateMethod}, {@link
UpdateValidatorMethod} or {@link SignalMethod} are processed like {@code current} is a
workflow interface even if it is not annotated with {@link WorkflowInterface} itself.
```

That'd make me think the default of `true` should work with non-workflow interfaces. But maybe I'm misunderstanding what was meant there.

Reply:
I think you are looking at the [wrong](https://github.com/temporalio/sdk-java/blob/95314a2357cf56efc1d703c657fd937bf6cbbc81/temporal-sdk/src/main/java/io/temporal/common/metadata/POJOWorkflowInterfaceMetadata.java#L131) method, the method I am calling is [here](https://github.com/temporalio/sdk-java/blob/95314a2357cf56efc1d703c657fd937bf6cbbc81/temporal-sdk/src/main/java/io/temporal/common/metadata/POJOWorkflowInterfaceMetadata.java#L108)